rollup-interface/src/state_machine/zk/mod.rs

```diff
@@ -61,20 +73,28 @@ pub trait ValidityCondition: Serialize + DeserializeOwned {
 /// if and only if the condition `validity_condition` is satisfied.
 ///
 /// The period of time covered by a state transition proof may be a single slot, or a range of slots on the DA layer.
-#[derive(Clone, Debug, Serialize, Deserialize)]
-pub struct StateTransition<C> {
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq)]
+pub struct StateTransition<C, Address> {
     /// The state of the rollup before the transition
     pub initial_state_root: [u8; 32],
     /// The state of the rollup after the transition
     pub final_state_root: [u8; 32],
+    /// The block hash of the state transition
+    pub slot_hash: [u8; 32],
+
+    /// Rewarded address
```

Comment:
Can we please elaborate a little bit beyond variable name and answer for what this address is rewarded.